thanks for the update! 
I see that some test are failing, haven't looked too much into it. 

```
Error: AssertionError: expected undefined to be 'John\'s' // Object.is equality
- Expected:
"John's"
+ Received:
undefined
 ❯ test/unit/SmilesDrawer.test.js:60:44
```
One more, while you're in here: the two width tests assert opts.width === 500, but 500 is the default  so they'd pass even if the attribute were ignored entirely. Use a non-default value to make them meaningful, e.g. '{"width": 1234}' / "{'width': 1234}" and assert toBe(1234).

--- a/test/unit/SmilesDrawer.test.js
+++ b/test/unit/SmilesDrawer.test.js
@@ -19,40 +19,46 @@ describe('SmilesDrawer', () => {
     });
 
     it('should parse valid JSON options', () => {
+        document.getElementById('target').remove();
         const element = document.createElement('div');
         element.setAttribute('data-smiles', 'C');
-        element.setAttribute('data-smiles-options', '{"width": 500}');
+        element.setAttribute('data-smiles-options', '{"width": 1234}');
         document.body.appendChild(element);
 
         const drawSpy = vi.spyOn(SmilesDrawer.prototype, 'draw');
+        drawSpy.mockClear();
         const drawer = new SmilesDrawer();
         drawer.apply('data-smiles');
 
         const instance = drawSpy.mock.instances[0];
-        expect(instance.drawer.opts.width).toBe(500);
+        expect(instance.drawer.opts.width).toBe(1234);
     });
 
     it('should parse JSON options with single quotes as fallback', () => {
+        document.getElementById('target').remove();
         const element = document.createElement('div');
         element.setAttribute('data-smiles', 'C');
-        element.setAttribute('data-smiles-options', "{'width': 500}");
+        element.setAttribute('data-smiles-options', "{'width': 1234}");
         document.body.appendChild(element);
 
         const drawSpy = vi.spyOn(SmilesDrawer.prototype, 'draw');
+        drawSpy.mockClear();
         const drawer = new SmilesDrawer();
         drawer.apply('data-smiles');
 
         const instance = drawSpy.mock.instances[0];
-        expect(instance.drawer.opts.width).toBe(500);
+        expect(instance.drawer.opts.width).toBe(1234);
     });
 
     it('should parse valid JSON containing an apostrophe without corruption', () => {
+        document.getElementById('target').remove();
         const element = document.createElement('div');
         element.setAttribute('data-smiles', 'C');
         element.setAttribute('data-smiles-options', '{"label": "John\'s"}');
         document.body.appendChild(element);
 
         const drawSpy = vi.spyOn(SmilesDrawer.prototype, 'draw');
+        drawSpy.mockClear();
         const drawer = new SmilesDrawer();
         drawer.apply('data-smiles');
 

